src/lib/error/api-error.ts

```diff
@@ -0,0 +1,270 @@
+import { v4 as uuidV4 } from 'uuid';
+import { FromSchema } from 'json-schema-to-ts';
+
+const UnleashApiErrorTypes = [
+    'OwaspValidationError',
+    'PasswordUndefinedError',
+    'InvalidTokenError',
+    'NoAccessError',
+    'UsedTokenError',
+    'InvalidOperationError',
+    'IncompatibleProjectError',
+    'OperationDeniedError',
+    'NotFoundError',
+    'NameExistsError',
+    'FeatureHasTagError',
+    'RoleInUseError',
+    'ProjectWithoutOwnerError',
+    'UnknownError',
+    'PasswordMismatch',
+    'PasswordMismatchError',
+    'DisabledError',
+    'ContentTypeError',
+    'NotImplementedError',
+
+    // server errors; not the end user's fault
+    'InternalError',
+] as const;
+
+// types that have extra data associated with them
+const UnleashApiErrorTypes2 = [
```

Comment:
I'd put the comment information into the const name itself. Adding number 2 seems to be obscure.

Reply:
Absolutely! I just forgot to rename it after hashing it out.